File: BE/src/main/java/com/capstone_project/hbts/request/BookingRequest.java
package com.capstone_project.hbts.request;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.sql.Timestamp;

@Data
@NoArgsConstructor
public class BookingRequest {

    private int bookedQuantity;

    private Timestamp bookingDate; // not required

    private Timestamp checkIn;

    private Timestamp checkOut;

    private int reviewStatus; // not required

    private int status; // not required

    private int hotelId;

    private int userId; // not required

}

File: travesily-fe-user-v1/src/app/_models/booking-request.ts
import {BookingDetail} from "./booking-detail";

export class BookingRequest {
  bookedQuantity: number;
  bookingDate: Date;
  checkIn: Date;
  checkOut: Date;
  reviewStatus: number;
  status: number;
  hotelId: number;
  userId: number
  bookingDetail: BookingDetail[]
  otherRequirement: string
  type: number
}
